src/lv_objx/lv_btnm.h

```diff
@@ -56,8 +56,15 @@ typedef struct
     uint16_t btn_cnt;                                 /*Number of button in 'map_p'(Handled by the library)*/
     uint16_t btn_id_pr;                               /*Index of the currently pressed button or LV_BTNM_BTN_NONE*/
     uint16_t btn_id_act;    /*Index of the active button (being pressed/released etc) or LV_BTNM_BTN_NONE */
-    uint8_t recolor : 1;    /*Enable button recoloring*/
-    uint8_t one_toggle : 1; /*Single button toggled at once*/
+#if LV_USE_GROUP
+    uint8_t recolor : 1,    /*Enable button recoloring*/
+            one_toggle : 1, /*Single button toggled at once*/
```

Comment:
@kisvegabor This is a neat way of writing out the bitfields, but we don't use it anywhere else. Do you think it matters whether it's used here or not?

Reply:
For some reason, I thought you have to write it that way for it to work. I could probably change it back tho